website/docs/r/organizational_unit.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
+layout: "dome9"
+page_title: "Check Point CloudGuard Dome9: dome9_organizational_unit"
+sidebar_current: "docs-resource-dome9-organizational-unit"
+description: |-
+  Create organizational unit in Dome9
+---
+
+# dome9_ruleset
+
+This resource is used to create and manage Organizational Unit in Dome9. An Organizational Unit is a group of cloud accounts representing, for example, a business unit or geographical region.
+
+## Example Usage
+
+Basic usage:
+
+```hcl
+resource "dome9_organizational_unit" "test_ou" {
+  name               = "some_organizational_unit"
```

Comment:
remove multi lines.
More recommended please run `terraform fmt -recursive` in the HCL section.